### ISSUE TITLE
IPv6 Mac Address Static Reservations Cannot Add

### ISSUE THREAD

**Issue by computeronix** [pi-hole/pi-hole#1993]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

1

---
**Expected behaviour:**

On the page, settings.php?tab=piholedhcp, I would expect to be able to add the decimal version of IPv6 IAID for each device and set a Mac Reservation for an IPv6 address here.

**Actual behaviour:**

On the page, settings.php?tab=piholedhcp, When I go to add the IAID to the Mac reservation table I receive the error invalid Mac Address.  The IAID is what the ID is shown in the DHCP Leases table above it.

**Steps to reproduce:**

1. Go to Settings > DHCP
2. Ensure IPv6 support is enabled (and IPv6 configured in the network properly)
3. In the DHCP reservation list attempt to add an IAID address.

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

5o5gp1vuro

**Troubleshooting undertaken, and/or other relevant information:**

Haven't found a workaround yet

**Comment by Tntdruid** [comment]
I made my own conf for that :)

myipv6.conf
dhcp-host=Mac,[ipv6adress],myhost

**Comment by dschaper** [comment]
@DL6ER This looks like it may fall under the `FTLDNS` realm.

**Comment by DL6ER** [comment]
I don't think this has changed with `FTLDNS`. I will hopefully have some time to look into issues end of the week.

**Comment by dschaper** [comment]
Ping for status. @DL6ER
